stone/target/python_rsrc/stone_serializers.py

```diff
@@ -229,7 +231,7 @@ def encode_primitive(self, validator, value):
         if isinstance(validator, bv.Void):
             return None
         elif isinstance(validator, bv.Timestamp):
-            return value.strftime(validator.format)
+            return _strftime(value, validator.format)
```

Comment:
I think so. Python 3 does not suffer from this limitation.

From https://docs.python.org/2/library/datetime.html#strftime-and-strptime-behavior:

> The exact range of years for which `strftime()` works also varies across platforms. Regardless of platform, years before 1900 cannot be used.

From https://docs.python.org/3/library/datetime.html#strftime-and-strptime-behavior:

> 2. The `strptime()` method can parse years in the full [1, 9999] range, but years < 1000 must be zero-filled to 4-digit width.
>
>     _Changed in version 3.2:_ In previous versions, `strftime()` method was restricted to years >= 1900.
>
>     _Changed in version 3.3:_ In version 3.2, `strftime()` method was restricted to years >= 1000.

Cf. http://bugs.python.org/issue1777412 (confirming that support of years < 1900 is a _feature_, not a bug, and would only be added to Python 3).